internal/pkg/cli/svc_init.go

```diff
@@ -231,6 +235,19 @@ func (o *initSvcOpts) newManifest() (encoding.BinaryMarshaler, error) {
 }
 
 func (o *initSvcOpts) newLoadBalancedWebServiceManifest() (*manifest.LoadBalancedWebService, error) {
+	copilotDirPath, err := o.ws.CopilotDirPath()
+	if err != nil {
+		return nil, fmt.Errorf("get copilot directory: %w", err)
+	}
+	wsRoot := filepath.Dir(copilotDirPath)
+	dfpath, err := filepath.Abs(o.DockerfilePath)
+	if err != nil {
+		return nil, err
+	}
+	o.DockerfilePath, err = filepath.Rel(wsRoot, dfpath)
+	if err != nil {
+		return nil, err
```

Comment:
nit: same here with wrapping

Reply:
One thing that I think we should throw an error for also is if the Dockerfile path is not under the `wsRoot`.
This is because in case the user uses a pipeline, and the Dockerfile is located outside of the git repository then we won't be able to build the container image

Reply:
I did this with strings.Contains; let me know if there's a better way!